portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -21,26 +21,52 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import java.util.ArrayList;
+import java.util.Arrays;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  Gson gson = new Gson();  
+private static Gson gson = new Gson();  
+private static final ArrayList<String> comments = new ArrayList<String>();  
   
  
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    ArrayList<String> jsonData = new ArrayList<String>();  
-    jsonData.add("nice!");
-    jsonData.add("cool!");
-    jsonData.add("awesome!");
-    String jsonOutput = convertToJsonUsingGson(jsonData);
     response.setContentType("application/json;");
-    response.getWriter().println(jsonOutput);
+    response.getWriter().println(convertToJsonUsingGson(comments));
   }
 
   private String convertToJsonUsingGson(ArrayList<String> jsonData) {
     return gson.toJson(jsonData);
   }
+  
+  // Get the comments from the user and display them
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String input = getParameter(request, "text-input", "");
+    String[] words = input.split("\\s*,\\s*");
```

Comment:
Why are you splitting up the words? I think you can just get the raw input with spaces and all and just save that (so you have the full comment as one entity). Otherwise, you'll have potentially multiple comments being split up into separate words, instead of grouped by how they were entered.

Reply:
Do you want to share the code that you wrote that wasn't compiling?

Reply:
Sorry, I'm just seeing this and I got it to compile but the comments aren't showing up. How should I share it with you?

Reply:
Instead of using the array called words, I tried to put the input directly into the comments ArrayList.

Reply:
I'm still thinking through it , I just realized why I can't do it like that